### ISSUE TITLE
[Site Request] hotcomics.me

### ISSUE THREAD

**Issue by xavimaa** [manga-download/hakuneko#6928]
**Name of the website**
HotComics

**Website urls (examples below)**
- Site: https://hotcomics.me/en
- Manga List: https://hotcomics.me/en/genres
- Manga example: https://hotcomics.me/en/downpour/D0wvg9KK.html
- Chapter online viewer example: https://hotcomics.me/en/downpour/episode-1-RfBxAttM.html

**Languages**
English

**Additional details**
UI design is really very close to Toomics

**Comment by MikeZeDev** [comment]
> UI design is really very close to Toomics

Not just UI i'd say ;)

**Comment by MikeZeDev** [comment]
Is that me or is language irrelevant? 

I keep getting eng content even if i change website language?

**Comment by xavimaa** [comment]
> Est-ce moi ou le langage n’est-il pas pertinent ?
> 
> Je continue à recevoir du contenu eng même si je change de langue de site Web ?

Yes I also tried to change the language to French, it only changes the language of the site but not the series.
